docs/run-tests-locally.md

```diff
@@ -16,19 +15,9 @@ You can run unit tests using your preferred IDE, or open a terminal in the proje
 dotnet test
 ```
 
-## Accessibility and UI tests
+## UI tests
```

Comment:
It would be nice to have a link to the automated test creation doc somewhere in this section to help tell people about it